### PR TITLE
use EnumArray for Verbal constants

### PR DESCRIPTION
**Pull request by bradallred**
## Checklist

- [x] Commit messages are descriptive and explain the rationale for changes
- [x] I used the same coding style as the surrounding code 
- [x] I have tested the proposed changes
- [x] I extended the documentation, if necessary
- [x] The proposed change builds also on our build bots (check after submission)

### CODE REVIEW

File: gemrb/core/Scriptable/Actor.cpp [hunk 1]
```diff
@@ -2078,19 +2077,19 @@ static void InitActorTables()
 
 	// verbal constant remapping, if omitted, it is an 1-1 mapping
 	// TODO: allow disabled VC slots
-	for (int i = 0; i < int(Verbal::LastVB); i++) {
-		VCMap[i] = static_cast<Verbal>(i);
+	for (auto i : EnumIterator<Verbal>()) {
+		VCMap[i] = i;
```
[the remark this answers is not in the document]
Comment (bradallred) [reply gemrb/core/Scriptable/Actor.cpp:2081]: the entire range

**Comment by bradallred** [comment]
ok, this seems to work now

**Comment by bradallred** [comment]
@lynxlynxlynx everything look good to you? is there anything beyond election/action sounds that needs to be tested?

**Comment by lynxlynxlynx** [comment]
It looks fine. A good test would include checking the sounds also in non-bg2, since that's what VCMap is for.

**Pull request merged**